static/styles/partials/_sizing.scss

```diff
@@ -37,4 +38,10 @@
 }
 .paypal-button {
   width: 11rem;
+}
+
+@media screen and (min-width: 30em) and (max-width: 60em) {
+  // .ts-subtext{
```

Comment:
If this commented out .ts-subtext isn't used, can we remove it

Reply:
Yep, definitely can.  Just removed it!